src/project-manager.ts

```diff
@@ -11,6 +11,10 @@ import { Logger, NoopLogger } from './logging';
 import { InMemoryFileSystem } from './memfs';
 import * as util from './util';
 
+const typeTypeScript = 'ts';
+const typeJavaScript = 'js';
+const types = [typeTypeScript, typeJavaScript];
```

Comment:
Instead of defining constants, I would define `type ConfigType = 'jsconfig' | 'tsconfig'`